app/src/main/java/fr/free/nrw/commons/profile/leaderboard/UserDetailAdapter.java

```diff
@@ -84,6 +86,17 @@ public void onBindViewHolder(@NonNull UserDetailAdapter.DataViewHolder holder, i
             holder.getContext().getResources().getString(R.string.count_prefix),
             leaderboardResponse.getCategoryCount()));
 
+        avatar.setOnClickListener(new DebouncingOnClickListener() {
+            @Override
+            public void doClick(final View v) {
+                new AlertDialog.Builder(v.getContext())
```

Comment:
Could you please add a unit test to cover this code? Thanks!

Reply:
Can you check your dependencies ? I just ran tests and it runs fine. 
@nicolas-raoul can we approve this PR and add tests in another PR ?

Reply:
Our policy for new pull requests is to require unit tests in the same pull request.

Reply:
Ok, thanks.
@ratneshnavlakhe let us know in case u face other errors. Also if you wish you can commit your draft test and we would be happy to take a look at it.